src/Hl7.Fhir.Support.Tests/JsonAssert.cs

```diff
@@ -19,8 +19,10 @@ public class JsonAssert
     {
         public static void AreSame(string filename, string expected, string actual, List<string> errors)
         {
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
Minor: I don't think these methods are marked `Obsolete` anymore, so these pragma's can be removed?